gallery/how-to/working-with-items/manual-download.md

```diff
@@ -1,72 +1,79 @@
 ﻿---
-ms.date:  06/26/18
+ms.date:  09/11/2018
 contributor:  JKeithB
 keywords:  gallery,powershell,psgallery
-title:  Manual Package Download 
+title:  Manual Package Download
 ---
 # Manual Package Download
 
-The Powershell Gallery supports downloading a package from the website directly, without using the PowerShellGet cmdlets. 
-The package will be downloaded as a NuGet package (.nupkg) file, which can then be easily copied to an internal repository. 
+The Powershell Gallery supports downloading a package from the website directly, without using the
+PowerShellGet cmdlets. The package will be downloaded as a NuGet package (.nupkg) file, which can
+then be easily copied to an internal repository.
 
 > [!NOTE]
-> Manual package download is **not** intended as a replacement for the Install-Module cmdlet. 
-> Downloading the package not install the module/script, and dependencies will not be downloaded. 
-> The instructions below provide the information needed to expand and deploy a PowerShell module as a reference, only. 
+> Manual package download is **not** intended as a replacement for the Install-Module cmdlet.
+> Downloading the package does not install the module or script. Dependencies are not included in
+> the NuGet package downloaded. The following instructions are provided for reference purposes
+> only. Deploying a package in this manner is unsupported.
```

Comment:
Delete last sentence. It is manual, and error prone, but we haven't said it's not supported.

Reply:
done